R/translate-sql-base.r

```diff
@@ -175,7 +175,42 @@ base_scalar <- sql_translator(
                         sql(ifelse(side == "both" | side == "right", "RTRIM(", "(")),
                         string
                         ,"))"
-                      )}
+                      )},
+
+  # lubridate functions
+
+  # SQL syntax reference links
+  # https://en.wikibooks.org/wiki/SQL_Dialects_Reference/Functions_and_expressions/Date_and_time_functions
+
+  # SQLite - https://www.sqlite.org/lang_datefunc.html
+  #        - https://www.sqlite.org/datatype3.html
+  #
+  as_date = sql_cast("DATE"),
```

Comment:
You should also include the base R `as.Date()` and `as.POSIXct()`, and Postgres also needs these equivalents.